### ISSUE TITLE
Support for number of unseen messages

### ISSUE THREAD

**Issue by dcokan** [tigase/tigase-server#26]
Hi, 

I am looking through your documentation and I cannot find the description of feature to see number of unseen messages in the group.

Looking into XEP standards I can see not approved yet standard XEP-0430 [(Inbox)](https://xmpp.org/extensions/xep-0430.html) But I can see the status of that is only Experimental... So I am wondering whether the feature will be only implemented in future or that is handled by other standard? 

Thanks, Dawid

**Comment by woj-tek** [comment]
I assume you refer to MUC (i.e. 'in the group')?

Inbox is indeed experimental and is not implemented, however we do support [XEP-0013: Flexible Offline Message Retrieval](https://xmpp.org/extensions/xep-0013.html) (which is in a way similar to Inbox)

**Comment by dcokan** [comment]
Ok, should work for me. Thanks :)

**Comment by dcokan** [comment]
Hi again, @woj-tek, Indeed if I installed a client and connected to server of tigase I installed on my infrastracture - the number of unread documents appears properly. However I am implementing the application in which chat is just a part and I need to obtain the unread count documents in other place too. Could you help me with pointing the right place i documentation so I can understand how to obtain these numbers? 

Thanks, Dawid

**Comment by woj-tek** [comment]
This highly depends on the (xmpp) library that you use. Specification defines XML stanza that can be used to obtain it here [2.2 Requesting Number of Messages](https://xmpp.org/extensions/xep-0013.html#request-number)

**Comment by dcokan** [comment]
Hi again,

I tried it out but the server doesn't respond with the number of messages to read. Instead I can find only empty <query> in response. 

I can see at point 2.1 of documentation -> https://xmpp.org/extensions/xep-0013.html#discover that requesting for supported feature should provide in response if xep-0013 is supported
> <feature var='http://jabber.org/protocol/offline'/>

But I don't get that in my response.. I checked for your documentation and added below to my configuration

> sess-man {
>    amp() {
>       'store-limit' = 200L
>    }
> }

But it didn't help. So is there anything else I should specify ? Thanks in advance
